src/lib/util.ts

```diff
@@ -10,7 +10,7 @@ import * as native from './native'
 
 import * as bcrypt from 'bcryptjs'
 
-export const random = (min, max) => Math.floor(Math.random() * (max - min)) + min;
+export const random = (min: number, max: number) => Math.floor(Math.random() * (max - min)) + min;
 interface propsType {
   [index: string]: any
```

Comment:
Is it actually completely untyped in Tauri?

Reply:
can you explain?

Reply:
This says that an object is expected that can hold anything as a value of a key. Then this interface is used in `Dialog.create()` call below. I have a strong suspicion it doesn't actually accept anything, but rather have a specific constraint on values (like strings) or maybe even has a specific requirements to the keys supported. In many cases you can "extract" the type of that parameter from the library too instead of resorting to `any` that erases all of the type information.

Reply:
cleaned up